pages/index.tsx

```diff
@@ -4,15 +4,17 @@
 // import { Button } from '../components/buttons';
 import { Layout } from '../containers/layout';
 import LandingPage from '../containers/landing/LandingPage';
+import { PageProps } from '../lib/types';
+import { withTranslation } from '../i18n';
 
-const Home = (): React.ReactElement => {
+const Home = ({ t }: PageProps): React.ReactElement => {
   // const handleClick = (): void => {
   //   const newLang = i18n.language === 'en' ? 'de' : 'en'
   //   i18n.changeLanguage(newLang)
   // }
 
   return (
-    <Layout title="Home" home>
+    <Layout title="Home" home t={t}>
```

Comment:
On the landing page I do not see the header that the layout creates

Reply:
Yes. That's because Camila used an additional header for the landing page rather than using the layout for it (which would have been preferable). In order for it not to look too bad layout now skips the header when "home" is set to true. In the long run, it would be good if the landing page just used the layout component, too (in which case the only difference would be that landing does not have a home button).